curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-fetch-promises-and-async-await-by-building-an-fcc-forum-leaderboard/6444062a33ab4c03d0c2e9f5.md

```diff
@@ -0,0 +1,193 @@
+---
+id: 6444062a33ab4c03d0c2e9f5
+title: Step 54
+challengeType: 0
+dashedName: step-54
+---
+
+# --description--
+
+The function named `showTime` that you created in the last couple of steps, you will use this to display the time since the last post was made. The function takes in a `time` string and returns a string with the time since the post was made. For example, if the post was made 5 minutes ago, the function will return `5m ago`. If the post was made 2 hours ago, the function will return `2h ago`. If the post was made 3 days ago, the function will return `3d ago`. To do this, run the function `showTime` with the argument `bumped_at` in the last `td` element of the `tr` element. 
```

Comment:
```suggestion
Use the `showTime` function, created in the previous steps, to display the time since the last post. This function takes a `time` string as input and returns a string representing the time since the post was made. For example, if the post was made 5 minutes ago, it will return `5m` ago. If the post was made 2 hours ago, it will return `2h` ago. If the post was made 3 days ago, it will return `3d` ago. To accomplish this, execute the `showTime` function with the `bumped_at` argument in the last `td` element of the `tr` element.
```

Reply:
added